pkg/engine/templates_generated.go

```diff
@@ -22898,6 +22898,14 @@ Set-AzureCNIConfig
     if ($IsAzureStack) {
         Add-Member -InputObject $configJson.plugins[0].ipam -MemberType NoteProperty -Name "environment" -Value "mas"
     }
+
```

Comment:
I was able to run `make generate` from inside the dev container (by running `make dev`).
It looks like there is some extra whitespace on line 22909 that is causing the CI failures.

Reply:
Thanks, @marosset. `make generate` after running `make dev` works. Updated the PR.